CHANGELOG.md

```diff
@@ -1,3 +1,126 @@
+### citus v11.1.0 (September 15, 2022) ###
+
+* Adds support for PostgreSQL 15beta4
+
+* Adds a rebalancer that uses background tasks for its execution
+
+* Adds `create_distributed_table_concurrently()` UDF to distribute tables
+  without blocking writes
+
+* Adds `citus_split_shard_by_split_points()` UDF that allows
+  splitting a shard to specified set of nodes without blocking writes
+  and based on given split points
+
+* Adds support for non-blocking tenant isolation
+
+* Adds support for isolation tenants that use partitioned tables
+  or columnar tables
+
+* Drops support for isolation tenants that use replicated tables
+
+* Separates columnar table access method into a separate logical extension
+
+* Adds support for logical replication in `replicate_reference_tables()`
```

Comment:
This sounds too technical, maybe: `Adds support for online replication in `replicate_reference_tables()``